pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -212,7 +212,7 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	if isSwapOn {
 		if failSwapOn {
-			return nil, fmt.Errorf("running with swap on is not supported, please disable swap! or set --fail-swap-on flag to false")
+			return nil, fmt.Errorf("running with swap on is not supported, please disable swap or set --fail-swap-on flag to false")
```

Comment:
Please fold this into the commit where this is first introduced.

Reply:
The reasoning for having this in a separate commit is that as [said above](https://github.com/kubernetes/kubernetes/pull/124060#discussion_r1601753544) this error message predated this PR, so in one commit I was just moving it around, and in another I was improving it. I thought it would be easier to understand this way and make every commit more concise.

In any case, this commit is now squashed. :+1: